modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -27,6 +27,38 @@ This illustration summarizes how each asset type (in green) appears in Exchange:
 
 image::ex2-exchange-assets.png[Illustration - How Assets appear in Exchange]
```

Comment:
No titles or numbers for any graphic elements. GLOBAL.

Reply:
Removed.